timur/docker-compose.yml

```diff
@@ -46,7 +45,7 @@ x-base-etna-services: &base-etna-services
       <<: *base-etna-env
       RUN_NPM_INSTALL: "true"
       SKIP_RUBY_SETUP: "true"
-    command: npm run webpack
+    command: npm run webpack-dev
 
```

Comment:
should probably be reverted? But I also have a weird kludge to convince monoetna to build my files in dev mode (futzing with package.json) - is there a canonical way this should be done @corps?

Reply:
Nah that's fine.  This is the correct place to setup webpack in dev mode.  We don't use docker-compose in production.  This is all dev setup.

In the past I think the reason I didn't use webpack-dev is that I saw massive spikes in build times due to the symbols and maps being built, but perhaps that depends on the app and webpack version (we have inconsistent versions).